src/commands/hld/reconcile.ts

```diff
@@ -264,8 +264,8 @@ const createIngressRouteForRing = (
     staticComponentPathInRing,
     "ingress-route.yaml"
   );
-  // TODO: figure out a way to grab the port from _somewhere_; store in bedrock.yaml?
-  const ingressRoute = TraefikIngressRoute(serviceName, ring, 8000, {
+  const k8sServicePort = serviceConfig.k8sServicePort ?? 80;
```

Comment:
Note. Because the `k8sServicePort` for `IBedrockServiceConfig` is possibly undefined. The default behaviour is to use port 80